src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -144,6 +155,21 @@ FUZZ_TARGET(coinselection)
             assert(result->GetWeight() > weight);
         }
     }
+
+    std::vector<COutput> manual_utxos;
+    CAmount manual_balance{CreateCoins(fuzzed_data_provider, manual_utxos, coin_params)};
+    if (manual_balance <= 0) return;
+    for (auto& result : results) {
+        if (!result) break;
+        auto manual_selection{ManualSelection(manual_utxos, manual_balance, coin_params.m_subtract_fee_outputs)};
```

Comment:
The manual selection is always the same. Could place it outside of the loop.

Reply:
Done